WordPress/Classes/ViewRelated/Blog/SiteSettingsViewController.m

```diff
@@ -811,41 +810,49 @@ - (void)showDeleteSiteForBlog:(Blog *)blog
 - (void)tableView:(UITableView *)tableView didSelectInAdvancedSectionRow:(NSInteger)row
 {
     switch (row) {
-        case SiteSettingsAdvancedStartOver: {
+        case SiteSettingsAdvancedStartOver:
             [self showStartOverForBlog:self.blog];
-        } break;
-        case SiteSettingsAdvancedDeleteSite: {
+            break;
+
+        case SiteSettingsAdvancedDeleteSite:
             [self showDeleteSiteForBlog:self.blog];
-        } break;
+            break;
     }
 }
 
 - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath *)indexPath
 {
+    [tableView deselectSelectedRowWithAnimation:YES];
+
     NSInteger settingsSection = [self.tableSections[indexPath.section] intValue];
     switch (settingsSection) {
-        case SiteSettingsSectionGeneral: {
+        case SiteSettingsSectionGeneral:
             [self tableView:tableView didSelectInGeneralSectionRow:indexPath.row];
-        } break;
-        case SiteSettingsSectionAccount: {
+            break;
+
+        case SiteSettingsSectionAccount:
             [self tableView:tableView didSelectInAccountSectionRow:indexPath.row];
-        } break;
-        case SiteSettingsSectionWriting: {
+            break;
+
+        case SiteSettingsSectionWriting:
             [self tableView:tableView didSelectInWritingSectionRow:indexPath.row];
-        } break;
-        case SiteSettingsSectionDiscussion: {
+            break;
+
+        case SiteSettingsSectionDiscussion:
             [self showDiscussionSettingsForBlog:self.blog];
-        } break;
-        case SiteSettingsSectionDevice: {
+            break;
+
+        case SiteSettingsSectionDevice:
             [self tableView:tableView didSelectInDeviceSectionRow:indexPath.row];
-        } break;
-        case SiteSettingsSectionRemoveSite:{
-            [tableView deselectSelectedRowWithAnimation:YES];
```

Comment:
Would probably have to restore `[tableView deselectSelectedRowWithAnimation:YES]` here if reverting back to standard behavior via `clearsSelectionOnViewWillAppear` as mentioned above on line `825`, or disable `selectionStyle` on the cell.